domain/consensus/processes/syncmanager/blocklocator.go

```diff
@@ -57,6 +52,20 @@ func (sm *syncManager) createBlockLocator(lowHash, highHash *externalapi.DomainH
 		nextBlueScore := currentBlockBlueScore - step
 		if currentBlockBlueScore < step {
 			nextBlueScore = lowBlockGHOSTDAGData.BlueScore()
+		} else if nextBlueScore < lowBlockGHOSTDAGData.BlueScore() {
+			nextBlueScore = lowBlockGHOSTDAGData.BlueScore()
+		}
```

Comment:
I added this if because now that we do it from PruningPoint and not genesis then we might have `nextBlueScore > lowBlockBlueScore && nextBlueScore > step` which will give us a one-off(because it will then break in the next loop iteration) block lower than lowBlueScore.